### ISSUE TITLE
[a11y] Columns should be made distinct from each other

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1320]
**Is your feature request related to a problem? Please describe.**
It is important to be able to discern between ranks on a tree. 

**Describe the solution you'd like**
Alternating colors in the background, solid line separators between columns.

![image](https://user-images.githubusercontent.com/37256050/164777095-d8ca698b-7ab5-48ea-a485-da5f629e7af8.png)
